src/App.jsx

```diff
@@ -2,13 +2,17 @@ import "./App.css";
 import { Header } from "./components/Header";
 import { BrowserRouter, Route, Routes } from "react-router-dom";
 import { ReviewsList } from "./components/ReviewsList";
+import { SingleReview } from "./components/SingleReview";
+import { NavBar } from "./components/NavBar";
 
 function App() {
   return (
     <BrowserRouter>
       <Header />
+      <NavBar />
       <Routes>
         <Route path="/" element={<ReviewsList />} />
+        <Route path="/:review_id" element={<SingleReview />} />
```

Comment:
This path is interesting, when a user reaches this point your url simply has a /(number) at the end, which doesnt really tell the user where they are on your webpage. My suggestion would be to add to this so it is more meaning full.